src/Microsoft.ML.StandardLearners/Standard/SdcaRegression.cs

```diff
@@ -43,8 +43,16 @@ public sealed class Options : OptionsBase
             /// <value>
             /// Defaults to <see cref="SquaredLoss"/>
             /// </value>
-            [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
-            public ISupportSdcaRegressionLossFactory LossFunction = new SquaredLossFactory();
+            [Argument(ArgumentType.Multiple, Name = "LossFunction", HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
+            internal ISupportSdcaRegressionLossFactory LossFunctionFactory = new SquaredLossFactory();
+
+            /// <summary>
+            /// A custom <a href="tmpurl_loss">loss</a>.
```

Comment:
>tmpurl_loss [](start = 34, length = 11)

Not sure what this is..

Reply:
see the line 41? same documentation as for the loss factory

---
In reply to: [262724768](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262724768) [](ancestors = 262724768)

Reply:
Then please fix both. Maybe you can git blame to find the killer.

---
In reply to: [262725105](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262725105) [](ancestors = 262725105,262724768)

Reply:
Shahab is working on this, issue #2356

---
In reply to: [262726212](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262726212) [](ancestors = 262726212,262725105,262724768)